cmd/inspector/main.go

```diff
@@ -131,9 +120,10 @@ type Inspector struct {
 // and the overlay cache
 func NewInspector(address, path string) (*Inspector, error) {
 	ctx := context.Background()
+
 	id, err := identity.Config{
-		CertPath: fmt.Sprintf("%s/ca.cert", path),
```

Comment:
Read these values from the config, don't rely on hardcoded values, because these can be overridden.

Reply:
This was only touched because it was named incorrectly, it had been changed to look for `identity.cert` and `identity.key`. I propose we leave this here and fix it elsewhere or in another PR if we want to change it.